### PR TITLE
QPACK: make TSS=0 a connection error

### LINKED ISSUE #1736: Is zero a valid argument for the Table State Synchronize instruction?
The QPACK draft is unclear about if insert count of zero should be considered a valid argument of the Table State Synchronize instruction.

I think we should better clarify that to avoid interoperability issues (my fear is that if some minor portion of QUIC stacks consider insert_count of zero to be invalid, some QUIC stacks might get shipped without noticing the existence of such stacks. Then, we would see interoperability issues in the wild).

FWIW, my weak preference goes to prohibiting use of zero as the argument. One way of achieving that would be to change the argument of the insruction to _number_of_inserts - 1_.

### PR DESCRIPTION
**Pull request by afrind**
Fixes #1736

### CODE REVIEW

File: draft-ietf-quic-qpack.md [hunk 1]
```diff
@@ -587,9 +587,10 @@ entries might cause a stream to become blocked, as described in
 ~~~~~~~~~~
 {:#fig-size-sync title="Table State Synchronize"}
 
-An encoder that receives an Insert Count that is greater than the number of
-dynamic table entries beyond the current Largest Known Received entry MUST treat
-this as a connection error of type `HTTP_QPACK_DECODER_STREAM_ERROR`.
+An encoder that receives an Insert Count equal to zero, or that is greater than
```
Comment (dtikhonov) [inline draft-ietf-quic-qpack.md]: I don't like the "that" in "or that:" does it refer to the insert count or to the encoder?

Comment (dtikhonov) [reply draft-ietf-quic-qpack.md]: Perhaps something along the lines of:

> The Insert Count value cannot be zero and it cannot be greater than the current Largest Known Received value.  An encoder that receives an invalid Insert Count value MUST treat it...

Comment (afrind) [reply draft-ietf-quic-qpack.md]: The constraint is tighter than < Largest Known Received, the text here captures it.  LKR + IC must be < Base Index

Comment (dtikhonov) [reply draft-ietf-quic-qpack.md]: OK, mine was just an example.  I'd split the sentence into two to make it unambiguous and easier to read.

**Pull request merged**